packages/@adobe/react-spectrum/src/index.ts

```diff
@@ -10,6 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
+'use client';
```

Comment:
couldn't we skip this if we checked for it in the optimizer same as the rest?

Reply:
unfortunately no, because this also re-exports some hooks which don't have it.